src/mjbots_hardware_interface.cpp

```diff
@@ -70,7 +83,11 @@ MjbotsHardwareInterface::MjbotsHardwareInterface(std::vector<std::shared_ptr<Joi
                                                  std::optional<::mjbots::pi3hat::Euler> imu_world_offset_deg,
                                                  bool dry_run,
                                                  bool print_torques,
-                                                 bool send_pd_commands)
+                                                 bool send_pd_commands,
+                                                 bool send_current,
+                                                 bool send_voltage,
+                                                 bool send_temperature,
+                                                 bool send_fault)
```

Comment:
maybe change bool names to request_XXXX except for send_pd_commands, which is actually a set of values.